app/migrations/Version20180223095718.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+declare(strict_types=1);
+
+namespace SWP\Migrations;
+
+use Behat\Transliterator\Transliterator;
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+use SWP\Bundle\ContentBundle\Model\RouteInterface;
+use Symfony\Component\DependencyInjection\ContainerAwareInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+class Version20180223095718 extends AbstractMigration implements ContainerAwareInterface
+{
+    /**
+     * @var ContainerInterface
+     */
+    private $container;
+
+    /**
+     * @param ContainerInterface|null $container
+     */
+    public function setContainer(ContainerInterface $container = null)
+    {
+        $this->container = $container;
+    }
+
+    public function up(Schema $schema)
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf('postgresql' !== $this->connection->getDatabasePlatform()->getName(), 'Migration can only be executed safely on \'postgresql\'.');
+
+        $entityManager = $this->container->get('doctrine.orm.default_entity_manager');
+        $routeRepository = $this->container->get('swp.repository.route');
+
+        /** @var RouteInterface $route */
+        foreach ($routeRepository->findBy(['slug' => null]) as $route) {
+            $route->setSlug(Transliterator::transliterate($route->getName()));
```

Comment:
It will be problematic if we will modify route entity. How about doing this with clean sql?

Reply:
It can be done with clean SQL but it will require some transliterate/slugify function to be executed on Postgres level explicitly. I would leave it like that for now as it does not break anything atm. Clean installation works fine too. We can discuss this further and see how it's solved in other projects in general as I saw it works fine for example in case of Sylius.

Reply:
You can use `Transliterator` but fetch rows with sql, and call sql update query for every row.